app/reducer.js

```diff
@@ -0,0 +1,33 @@
+
+// Lets us use the combine Reducers function at the bottom instead of
+// individually exporting each function
+// Its a property of an object, hence the notation
+import { combineReducers } from 'redux';
+
+
+function seriesInfo(state, action) {
+  if (action.type === 'SERIES_INFO@LOAD') {
```

Comment:
For your reducers, prefer a `switch` statement instead of individual if statements.

And for the default condition, return the old state.
This way if an action comes in that should not change the current part of the state, then nothing will happen.